src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/whitespaceafter/InputWhitespaceAfterLiteralReturn.java

```diff
@@ -0,0 +1,17 @@
+/*
+WhitespaceAfter
+tokens = LITERAL_RETURN
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.whitespaceafter;
+
+public class InputWhitespaceAfterLiteralReturn {
+    public int testOne (int x){
+        return (x * x); // OK
+    }
+    public int testTwo (int x){
```

Comment:
please use correct spacing for method signature. Same for method above
`testTwo(int x) {`

Reply:
Changes Done